### PR TITLE
add import section for polyline importer

### PR DESCRIPTION
**Pull request by missinglink**
add import section for polyline importer

see: https://github.com/pelias/polylines

### CODE REVIEW

File: config/defaults.json [hunk 1]
```diff
@@ -58,6 +58,10 @@
       "datapath": "/mnt/pelias/openaddresses",
       "files": []
     },
+    "polyline": {
+      "datapath": "/mnt/pelias/polyline",
```
Comment (orangejulius) [inline config/defaults.json]: Add `"adminLookup": false,` here. It won't change anything obviously, but people outside Mapzen look at this file for a list of config options that can be changed.

Comment (missinglink) [inline config/defaults.json]: I just copied the OA one, I'll add it in for both.

Comment (orangejulius) [inline config/defaults.json]: hah, I didn't realize it wasn't there for OA. thanks for fixing

**Comment by orangejulius** [comment]
LGTM

**Comment by dianashk** [comment]
:shipit:

**Pull request merged**